### DISCUSSION TITLE
PaddleOCR/latest/ppocr/model_train/detection

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14232] [Q&A]
# PaddleOCR/latest/ppocr/model_train/detection

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_train/detection.html

**Comment by laochi-love-cmy1314** [comment]
请问模型导出后没有.pdmodel文件和.pdiparams.info文件，反而多了inference.json文件是什么原因呢，模型导出命令中使用的就是训练用的配置文件和训练之后的best_accuracy

**Reply by GreatV** [reply]
@laochi-love-cmy1314 这个是新的pir导出格式，如果想回到旧版本可以使用 <= paddle3.0 beta 1.0 版本的paddle。也可以直接使用新版本的模型。

**Reply by laochi-love-cmy1314** [reply]
请问如何使用最新版本呢？我使用PaddleOCR（lang="ch",det_model_dir="",cls_model_dir="",rec_model_dir=）这个函数，设置了三个模型目录，按官网之前的那种是可以的，但是我把目录换成这种新导出的模型目录去识别的时候，会先下载缺失的.pdmodel文件和.pdiparams.info文件，这种应该不是使用我自己训练的模型吧

**Reply by GreatV** [reply]
可以先切换到旧版本，新版本可能还在适配中。 https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/develop/install/pip/linux-pip.html 根据自己的需要按照对应的paddle版本然后 把 3.0.0b2 改成3.0.0b1

**Reply by laochi-love-cmy1314** [reply]
好的哥，非常感谢

**Comment by Zhou2518** [comment]
python3 tools/train.py -c configs/det/det_mv3_db.yml \-o Global.pretrained_model=./pretrain_models/MobileNetV3_large_x0_5_pretrained
时报错
ValueError: (InvalidArgument) Broadcast dimension mismatch. Operands could not be broadcast together with the shape of X = [64, 256, 25, 25] and the shape of Y = [64, 256, 26, 26]. Received [25] in X is not equal to [26] in Y at i:2.
  [Hint: Expected x_dims_array[i] == y_dims_array[i] || x_dims_array[i] <= 1 || y_dims_array[i] <= 1 == true, but received x_dims_array[i] == y_dims_array[i] || x_dims_array[i] <= 1 || y_dims_array[i] <= 1:0 != true:1.] (at ../paddle/phi/kernels/funcs/common_shape.h:86)

是环境安装不对么？
